src/components/common/SkillsList.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="w-full flex flex-col gap-3 font-montserrat">
+    <span class="font-semibold text-gray-700 text-xl max-xl:text-base max-md:text-sm">
+      Skills:
+    </span>
+    <ul v-if="skills.length">
```

Comment:
In this case, I recommend to use just div

Element div not allowed as child of element ul in this context

you can check it here
https://validator.w3.org/nu/#textarea

Reply:
Thank you, I checked it. Will be fixed